### PR TITLE
Hub/cosmetics

### PR DESCRIPTION
**Pull request by jordanlesich**
## GitHub Issue

NA

## Changes

- Remove toggle button in mobile
- Automatically render cards in mobile 
- Fall back logo/image on the DAO cards
- Hide the Go button

## Packages Added

NA

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/hub-app/src/components/DaoCard.tsx [hunk 1]
```diff
@@ -92,9 +93,9 @@ export const DaoCard = ({
         <Tag>{networkId}</Tag>
         <Tag>{contractType}</Tag>
       </div>
-      <div>
+      {/* <div>
         <Button secondary>Go</Button>
-      </div>
+      </div> */}
```
Comment (0xRowdy) [inline apps/hub-app/src/components/DaoCard.tsx:98]: Delete? Add a comment if you want to keep for future?

File: apps/hub-app/src/components/TableControl.tsx [hunk 2]
```diff
@@ -79,14 +80,16 @@ const TableControl = ({
...
       />
-      <Button
-        secondary
-        onClick={toggleListType}
-        IconLeft={IconGrid}
-        className="list-type-toggle"
-      >
-        {listType === 'table' ? 'Card View' : 'List View'}
-      </Button>
+      {isMobile || (
```
Comment (0xRowdy) [inline apps/hub-app/src/components/TableControl.tsx:83]: Was this supposed to be the && ?

**Review by 0xRowdy — APPROVED** [review]

File: apps/hub-app/src/components/DaoCard.tsx (hunk 1 above)
Comment (jordanlesich) [reply apps/hub-app/src/components/DaoCard.tsx:98]: Do want to keep for future. Good catch though

File: apps/hub-app/src/components/TableControl.tsx (hunk 2 above)
Comment (jordanlesich) [reply apps/hub-app/src/components/TableControl.tsx:83]: This one's weird.

**Pull request merged**